pom.xml

```diff
@@ -17,20 +17,50 @@
     <parent>
         <artifactId>jberet-parent</artifactId>
         <groupId>org.jberet</groupId>
-        <version>1.4.0.Alpha-SNAPSHOT</version>
+        <version>2.0.3.Final</version>
     </parent>
```

Comment:
the version should be a SNAPSHOT version till it's released. If you want to keep syn with the latest jberet-core version, that's fine. The latest jberet-core release is 2.1.0.Final. So this version here should be 2.1.0.Final-SNAPSHOT. But in the future the versions will diverge.